themes/default/content/blog/google-cloud-7-0/index.md

```diff
@@ -0,0 +1,211 @@
+---
+title: "Pulumi Google Cloud Classic 7.0"
+date: 2023-11-05T09:00:00-04:00
+draft: false
+meta_desc: "The Latest for Your Modern Cloud Infrastructure: Pulumi Google Cloud Provider 7.0."
+meta_image: meta.png
+authors:
+  - monica-rodriguez
+tags:
+  - google-cloud
+  - releases
+
+---
+
+The latest major release of the Pulumi Google Cloud Classic Provider is available now! This [7.0 release](https://github.com/pulumi/pulumi-gcp/releases/tag/v7.0.0) contains the latest upstream changes to keep you up-to-date along with a highly requested bug fix, keeping your journey in managing Google Cloud resources fresh and smooth.
+
+The Pulumi Google Cloud Classic provider can be used to provision any of the cloud resources available in the upstream provider. It is part of the suite of [Pulumi official providers](https://www.pulumi.com/docs/intro/cloud-providers/), which means that it is officially maintained and supported by Pulumi. The provider is also open source and available on [GitHub](https://github.com/pulumi/pulumi-gcp) for you to contribute to and improve.
+
+{{< toc >}}
+
+<!--more-->
+
+## Looking back
+
+It's been almost 2 years since the last major release of the Google Cloud provider, version 6.0, which was released in November 2021. Since then, we've been hard at work improving the provider and adding new features. Here are some of the highlights:
```

Comment:
maybe something like ... "even though we've delivered over 150 minor updates" or any bit of prose that doesn't make it look like we've done nothing for 2yrs!

Reply:
Updated per feedback.